go.mod

```diff
@@ -5,6 +5,7 @@ go 1.16
 require (
 	github.com/crossplane/crossplane v1.1.0
 	github.com/crossplane/crossplane-runtime v0.13.0
+	gopkg.in/yaml.v2 v2.3.0
```

Comment:
nitpick: can we use `sigs.k8s.io/yaml`, which is slightly different here for consistency with other repos?

ref: https://github.com/crossplane/crossplane/blob/d8f2b4f86a634aad19af127c48333e14ada227b8/go.mod#L26